http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -146,6 +148,8 @@ private SslContext sslContext(String scheme) {
         if (scheme.equalsIgnoreCase("https")) {
             SslContextBuilder builder = SslContextBuilder.forClient().sslProvider(defaultClientProvider());
             if (configuration.trustAllCertificates()) {
+                log.warn(() -> "SSL Certificate verification is disabled. This is not a safe setting and should only be "
```

Comment:
Should we use the same strategy for Netty's cert checking?

Reply:
Its almost similar, we use the attributemap to create NettyConfiguration class which is an internal API. So customers can only disable certs through buildWithDefaults method (same as in Apache)

Reply:
https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java#L76

https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java#L32

Reply:
Can we get rid of this then? 

https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java#L221